LICENSE

```diff
@@ -1,8 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2020, Board of Trustees of Columbia University in the
-city of New York and Brookhaven Science Associates, Brookhaven National
-Laboratory.
+Copyright (c) 2020, Board of Trustees of Columbia University in the city of New York and Brookhaven Science Associates, Brookhaven National Laboratory.
```

Comment:
This should read:
"Copyright (c) 2020, Board of Trustees of Columbia University in the city of New York"
NO Brookhaven here please